encoder.go

```diff
@@ -34,9 +34,76 @@ bridge_encoder_get_sample_rate(OpusEncoder *st)
 	opus_encoder_ctl(st, OPUS_GET_SAMPLE_RATE(&sample_rate));
 	return sample_rate;
 }
+
+
+int
+bridge_encoder_set_bitrate(OpusEncoder *st, opus_int32 bitrate)
+{
+	int res;
+	res = opus_encoder_ctl(st, OPUS_SET_BITRATE(bitrate));
+	return res;
+}
+
+int
+bridge_encoder_get_bitrate(OpusEncoder *st, opus_int32 *bitrate)
+{
+	int res;
+	res = opus_encoder_ctl(st, OPUS_GET_BITRATE(bitrate));
+	return res;
+}
+
+int
+bridge_encoder_set_complexity(OpusEncoder *st, opus_int32 complexity)
+{
+	int res;
+	res = opus_encoder_ctl(st, OPUS_SET_COMPLEXITY(complexity));
+	return res;
+}
+
+int
+bridge_encoder_get_complexity(OpusEncoder *st, opus_int32 *complexity)
+{
+	int res;
+	res = opus_encoder_ctl(st, OPUS_GET_COMPLEXITY(complexity));
+	return res;
+}
+
+int
+bridge_encoder_set_max_bandwidth(OpusEncoder *st, opus_int32 max_bw)
+{
+	int res;
+	res = opus_encoder_ctl(st, OPUS_SET_MAX_BANDWIDTH(max_bw));
+	return res;
+}
+
+int
+bridge_encoder_get_max_bandwidth(OpusEncoder *st, opus_int32 *max_bw)
+{
+	int res;
+	res = opus_encoder_ctl(st, OPUS_GET_MAX_BANDWIDTH(max_bw));
+	return res;
+}
+
 */
 import "C"
 
+const (
+	// Auto bitrate
+	Auto = C.OPUS_AUTO
+	// Maximum bitrate
+	BitrateMax = C.OPUS_BITRATE_MAX
```

Comment:
Is this the maximum bitrate? Then MaxBitrate is a better name, like `math.MaxInt` for example.